app/models/status.rb

```diff
@@ -284,14 +284,21 @@ def carried_over_reply_to_account_id
   end
 
   def find_statuses_from_tree_path(ids, account)
-    statuses = Status.where(id: ids).to_a
+    statuses = Status.where(id: ids).includes(:account).to_a
+
+    # FIXME: n+1 bonanza
```

Comment:
No it's been present since day 1, it's a fix for another day with less time pressure